include/internal/ktls.h

```diff
@@ -60,7 +73,12 @@ static ossl_inline int ktls_start(int fd,
         return setsockopt(fd, IPPROTO_TCP, TCP_TXTLS_ENABLE,
                           tls_en, len) ? 0 : 1;
     else
```

Comment:
It might be better just to remove this "else" - the "if" block "returns" anyway.

Reply:
Done